apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -17,6 +17,8 @@ import {
 import ValidationStep, {Status} from '../../../ui/ValidationStep';
 import {BOARD_TYPE} from '../CircuitPlaygroundBoard';
 import experiments from '@cdo/apps/util/experiments';
+import _ from 'lodash';
+import yaml from 'js-yaml';
```

Comment:
Tiny nit: could make the manifest a JSON file so we don't need to pull in a library to parse it.